src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -1080,8 +1080,7 @@ PeerImp::onMessage (std::shared_ptr <protocol::TMProposeSet> const& m)
 {
     protocol::TMProposeSet& set = *m;
 
-    if (overlay_.setup().expire &&
-            set.has_hops() && ! slot_->cluster())
+    if (set.has_hops() && ! slot_->cluster())
```

Comment:
We could have used better internal documentation but this is not a public feature so we should not advertise it to users.